cmd/swapd/main.go

```diff
@@ -276,9 +275,10 @@ func setLogLevels(level string) {
 }
 
 func runDaemon(c *cli.Context) error {
-	ctx, cancel := context.WithCancel(c.Context)
-	defer cancel()
-	go signalHandler(ctx, cancel)
+	// Fail if any non-flag arguments were passed
+	if c.Args().Present() {
+		return fmt.Errorf("unknown command %q", c.Args().First())
+	}
```

Comment:
I guess this is only an issue if the whole app runs off of the top level executable with no subcommands. `swapcli` automatically handles unknown subcommands.